dex-template/__DexName__-integration.test.ts.template

```diff
@@ -0,0 +1,83 @@
+import dotenv from 'dotenv';
+dotenv.config();
+
+import { DummyDexHelper } from '../../dex-helper/index';
+import { Network, SwapSide } from '../../constants';
+import { __DexName__ } from './__DexNameParam__';
+import { checkPoolPrices, checkPoolsLiquidity } from '../../../tests/utils';
+import { Tokens } from '../../../tests/constants-e2e';
+
+/*
+  README
+  ======
+
+  This test script adds tests for __DexName__ general integration 
+  with the DEX interface. The test cases below are example tests. 
+  It is recommended to add tests which cover __DexName__ specific
+  logic. 
+
+  You can run this individual test script by running:
+  `npx jest src/dex/<dex-name>/<dex-name>-integration.tests.ts`
+
+  (This comment should be removed from the final implementation)
+*/
+
+const TokenASymbol = 'TokenASymbol';
+const TokenA = Tokens[TokenASymbol];
+
+const TokenBSymbol = 'TokenBSymbol';
+const TokenB = Tokens[TokenBSymbol];
+
+const amounts = [
+  BigInt('0'),
+  BigInt('1000000000000000000'),
+  BigInt('2000000000000000000'),
+];
+
+const dexKey = '__DexName__';
+
+describe('__DexName__', function () {
+  it('getPoolIdentifiers and getPricesVolume', async function () {
```

Comment:
Should have a BUY version of this test as well...

Reply:
fixed